scripts/deploy/package.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Consider setting a minimum [engines](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#engines) node/npm version (either same as cht core or node 18+) mostly to set expectations but would also be good to run tests explicitly against all the supported major LTS versions

Consider adding [bin](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#bin) and ensure it is executable using [npx](https://docs.npmjs.com/cli/v7/commands/npx) and similar runtimes, ideally there should also be a publish build step added, but that can be out of scope for this PR, as-is it is already better than the old solution IMO

Reply:
handled.